### PR TITLE
Add scripts for installer

### PR DESCRIPTION
**Pull request by suihkulokki**
Carry the scripts needed for install grub on SD card installer and
flashing content from the installer to hikey.

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,10 @@
+96boards-tools (0.8) jessie; urgency=medium
+
+  * Add script for setting up grub for hikey
+  * Add flash-hikey script for calling from isntaller
```
Comment (fboudra) [inline debian/changelog]: typo

File: flash-hikey [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+# This script is called by the graphical installer, and:
+#  * it is called from the folder which contains the build to flash
+#  * with "-o /dev/<output>"
+
+OUTPUT=$(basename $2)
+zcat $PWD/boot.img.gz  |sudo dd of=/dev/${OUTPUT}p6 bs=1M
```
Comment (fboudra) [inline flash-hikey]: suggest to use: bs=4M oflag=sync status=noxfer
to be consistent with our wiki/doc recommendations.

File: grub-install-hikey [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+if [ $sdcard -eq 1 ]
+then
+    conf=/usr/share/96boards-tools/grub/grub-sd.configfile
+    root='(hd1,msdos2)'
+    rootdev=/dev/mmcblk1p2
+else
+    conf=/usr/share/96boards-tools/grub/grub.configfile
+    root='(hd0,gpt9)'
+    rootdev=/dev/mmcblk0p9
```
Comment (ndechesne) [inline grub-install-hikey]: it seems very fragile to hard code that here. do we really want that? i don't have the entire context of this patch.. but hard coding partition number is error prone

File: grub-install-hikey [hunk 4]
```diff
@@ -0,0 +1,79 @@
+#!/bin/bash -e
+# script to install/update grub on hikey
+# supports only install to eMMC or SD
+
```
Comment (ndechesne) [inline grub-install-hikey]: should we check that we run as  uid 0 before doing anything?

File: flash-hikey (hunk 2 above)
Comment (ndechesne) [inline flash-hikey]: sudo probably not needed, as this is likely to be required to run as root already, no? otherwise that requires user to input passwd

**Comment by fboudra** [comment]
Ping, @suihkulokki can you follow up on the comments?

Should we get this pull request merged before https://review.linaro.org/#/c/13018/ or it doesn't matter?

**Comment by suihkulokki** [comment]
@fboudra order of patches doesn't matter. I'll provide an updated patch soon

**Comment by fboudra** [comment]
Ack. Thanks! I'll have a look to the build job then.

File: grub-install-hikey (hunk 3 above)
Comment (suihkulokki) [inline grub-install-hikey]: For sd card we should hardcode, we don't want to load anything from the (potentially corrupt or empty) eMMC. The eMMC else bit there is not used, it is more for documentation/recovery .

File: flash-hikey (hunk 2 above)
Comment (suihkulokki) [inline flash-hikey]: @ndechesne the sudo is needed, this is called from the installer gui:
https://git.linaro.org/people/nicolas.dechesne/db-boot-tools.git/blob/ce8fc4d5208523776c166e7d28fecfc7c7a4df54:/flash

**Pull request merged**